python/pyspark/sql/session.py

```diff
@@ -136,7 +136,7 @@ def appName(self, name):
         @since(2.0)
         def enableHiveSupport(self):
             """Enables Hive support, including connectivity to a persistent Hive metastore, support
-            for Hive serdes, and Hive user-defined functions.
+            for Hive SerDe's, and Hive user-defined functions.
```

Comment:
? This should not be a possessive form. Please revert this change.

Reply:
SerDes is fine though